components/gitpod-cli/cmd/init.go

```diff
@@ -52,15 +52,138 @@ Create a Gitpod configuration for this project.
 			log.Fatal(err)
 		}
 		if !interactive {
-			d = []byte(`# List the start up tasks. Learn more https://www.gitpod.io/docs/config-start-tasks/
-tasks:
-  - init: echo 'init script' # runs during prebuild
-    command: echo 'start script'
-
-# List the ports to expose. Learn more https://www.gitpod.io/docs/config-ports/
-ports:
-  - port: 3000
-    onOpen: open-preview
+			d = []byte(`## Learn more about this file at 'https://www.gitpod.io/docs/references/gitpod-yml'
+
+## This '.gitpod.yml' file when placed at the root of a project instructs
+## Gitpod how to prepare & build the project, start development environments
+## and configure continuous prebuilds. Prebuilds when enabled builds a project
+## like a CI server so you can start coding right away - no more waiting for
+## dependencies to download and builds to finish when reviewing pull-requests
```

Comment:
suggestion (non-blocking): remove the dash

It seems like GitHub Docs point to the feature just being called `pull requests`

```suggestion
## dependencies to download and builds to finish when reviewing pull requests
```